src/components/search/search.tsx

```diff
@@ -29,20 +31,36 @@ const Search = () => {
 
   return (
     <>
-      <SearchBar
-        id="searchbar"
-        invalidText="Could not find the item"
-        labelText="Search Digital Collections"
-        onSubmit={(event) => handleSubmit(event, router)}
-        textInputProps={{
-          labelText: "Search keyword(s)",
-          name: "textInputName",
-          onChange: handleTextChange,
-          value: keywords,
-          placeholder: "Search keyword(s)",
+      <Box
+        id="search-wrapper"
+        sx={{
+          alignItems: "start",
+          width: "100%",
+          "@media screen and (max-width: 768px)": {
```

Comment:
Noting another instance of max-width that should be written as min-width if possible.